### DISCUSSION TITLE
Move Cache to another device

### DISCUSSION THREAD

**Discussion by juliothomaz** [abraunegg/onedrive#2286] [Q&A] [answered]
Hi,

Please, I have low space available on the Debian main mount partition (42Gb on SSD), so when a try to move the .cache to another device (380Gb Ext4 on HDD) i get the following message:

Could not move cache for mount: ( .... ) : invalid cross-device link 

Is there anything could be done?

Thank you very much!

**Comment by abraunegg** [comment, marked as answer]
@juliothomaz 


> Hi,
> 
> Please, I have low space available on the Debian main mount partition (42Gb on SSD), so when a try to move the .cache to another device (380Gb Ext4 on HDD) i get the following message:
> 
> Could not move cache for mount: ( .... ) : invalid cross-device link
> 
> Is there anything could be done?
> 
> Thank you very much!

This client does not use a '.cache' folder. It uses your user's '.config' folder.

As your question really relates to Debian - you are better off asking on Debian help forums than here to solve your problem.
